### PR TITLE
fix: ready to publish to NPM

### PR DESCRIPTION
**Pull request by chiefcll**

### CODE REVIEW

File: components/Announcer/Announcer.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ This class does its best to work around these issues, but speech synth api can r
 Extend your application with `Announcer` before boot:
 
 ```js
-import { Announcer } from '@lightning/ui'
+import { Announcer } from '@lightning/ui-components'
```
Comment (chelseasimek) [inline components/Announcer/Announcer.mdx:21]: this needs to be @lightningjs/ui-components

File: components/Keyboard/Keyboard.mdx [hunk 2]
```diff
@@ -9,7 +9,7 @@ On screen keyboard component.
 You can import and use the `KEYBOARD_FORMATS` object to get pre-formatted and sized keyboards out of the box, or you can pass a custom `formats` object.
 
 ```js
-import { Keyboard, KEYBOARD_FORMATS } from '@lightning/ui';
+import { Keyboard, KEYBOARD_FORMATS } from '@lightning/ui-components';
```
Comment (chelseasimek) [inline components/Keyboard/Keyboard.mdx:12]: this needs to be @lightningjs/ui-components

File: package.json [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {
-  "name": "lightning-ui-components",
-  "version": "0.0.0",
+  "name": "@lightningjs/ui-components",
+  "version": "0.0.1",
```
Comment (chelseasimek) [inline package.json:3]: should we bump this to 1.0.0 before we publish?

**Review by chelseasimek — CHANGES_REQUESTED** [review]
import names in docs need to be updated still

File: package.json (hunk 3 above)
Comment (chiefcll) [reply package.json:3]: Once both PRs are in I will.

**Review by chelseasimek — APPROVED** [review]

**Pull request merged**